MM2SpaceCenter.roboFontExt/lib/MM2SpaceCenter.py

```diff
@@ -15,139 +15,147 @@
 from defconAppKit.windows.baseWindow import BaseWindowController
 from mojo.events import addObserver, removeObserver, postEvent
 from vanilla import *
+from lib.UI.integerEditText import NumberEditText
 from mojo.extensions import *
 import codecs
 
 class MM2SpaceCenter:
     
     def activateModule(self):
         addObserver(self, "MMPairChangedObserver", "MetricsMachine.currentPairChanged")
+        
         print ('MM2SpaceCenter activated')
 
+
```

Comment:
I added/removed some line breaks for a bit of consistency and to tighten the gaps so I could see indent relationships better. Hope that's ok!